testing/syndication/tmpgkkoqykx

```diff
@@ -0,0 +1 @@
+{"filepath_blacklist": {"jsx": "1", "xml": "1", "html": "1"}, "autoclose_preview_window_after_insertion": 0, "collect_identifiers_from_tags_files": 0, "server_keep_logfiles": 0, "filetype_blacklist": {"notes": "1", "markdown": "1", "netrw": "1", "unite": "1", "pandoc": "1", "tagbar": "1", "qf": "1", "vimwiki": "1", "text": "1", "infolog": "1", "mail": "1"}, "key_list_previous_completion": ["<S-TAB>", "<Up>"], "seed_identifiers_with_syntax": 0, "auto_stop_csharp_server": 1, "show_diagnostics_ui": 1, "collect_identifiers_from_comments_and_strings": 0, "complete_in_strings": 1, "log_level": "info", "filetype_specific_completion_to_disable": {"gitcommit": "1"}, "open_loclist_on_ycm_diags": 1, "extra_conf_vim_data": [], "keep_logfiles": 0, "min_num_of_chars_for_completion": 2, "gocode_binary_path": "", "max_diagnostics_to_display": 30, "enable_diagnostic_highlighting": 1, "max_num_identifier_candidates": 10, "key_detailed_diagnostics": "<leader>d", "warning_symbol": ">>", "use_ultisnips_completer": 1, "rust_src_path": "", "auto_trigger": 1, "goto_buffer_command": "same-buffer", "echo_current_diagnostic": 1, "autoclose_preview_window_after_completion": 0, "csharp_server_port": 0, "confirm_extra_conf": 1, "racerd_binary_path": "", "python_binary_path": "", "java_jdtls_use_clean_workspace": 1, "key_list_select_completion": ["<TAB>", "<Down>"], "server_python_interpreter": "", "max_num_candidates": 50, "disable_for_files_larger_than_kb": 1000, "always_populate_location_list": 0, "complete_in_comments": 0, "add_preview_to_completeopt": 0, "global_ycm_extra_conf": "", "auto_start_csharp_server": 1, "filepath_completion_use_working_dir": 0, "enable_diagnostic_signs": 1, "extra_conf_globlist": [], "key_invoke_completion": "<C-Space>", "cache_omnifunc": 1, "filetype_whitelist": {"*": "1"}, "min_num_identifier_candidate_chars": 0, "key_list_stop_completion": ["<C-y>"], "error_symbol": ">>", "godef_binary_path": "", "semantic_triggers": {}, "hmac_secret": "3AIqfqOLdiFQlX9gvNKaKg=="}
```

Comment:
Did you mean to commit this and the log files?

Reply:
I did not, will remove